frappe/core/doctype/submission_queue/submission_queue.json

```diff
@@ -102,6 +103,11 @@
    "report": 1,
    "role": "System Manager",
    "share": 1
+  },
+  {
+   "if_owner": 1,
+   "read": 1,
+   "role": "All"
```

Comment:
added permission for all users who're owners to read the submission queue documents